Gemfile.lock

```diff
@@ -2,15 +2,15 @@ GIT
   remote: https://github.com/macdiesel/mongoid-tree
   revision: b381dd56f1b3b061df8f4b4181d5440dea1602d1
   specs:
-    mongoid-tree (2.0.0)
-      mongoid (>= 4.0, <= 5.0)
+    mongoid-tree (2.1.1)
```

Comment:
The gemfile is using a custom fork.  Please update the Gemfile to use `gem 'mongoid-tree', :require => 'mongoid/tree'` rather than `gem 'mongoid-tree', :git => 'https://github.com/macdiesel/mongoid-tree'`

Reply:
@bderusha I changed it.  Getting this error 
```
The following gems are missing from the
DEPENDENCIES section: 'zeitwerk' 'concurrent-ruby'
```
I tried adding these dependancies but it turns out that zeitwerk 2.2 is not compatible with ruby 2.4.1 and that is the minimum version of zertwerk that is compatible with activesupport 6.0.2.1 that we are using. I am guessing maybe this was the reason they were using the forked repository.  We need to upgrade ruby to 2.4.4 to make it compatible. Is that feasible to do?

Reply:
Let's try it and see.  If the ruby change requires a lot of refactoring, check-in before doing it.  But let's try it.